pkgs/applications/networking/instant-messengers/teamspeak/client.nix

```diff
@@ -114,43 +161,45 @@ stdenv.mkDerivation rec {
       url = "https://www.teamspeak.com/en/privacy-and-terms/";
       free = false;
     };
-    maintainers = with maintainers; [ lhvwb lukegb atemu ];
+    maintainers = with maintainers; [
+      lhvwb
+      lukegb
+      atemu
+    ];
     mainProgram = "ts3client";
     platforms = [ "x86_64-linux" ];
   };
 }
 
 /*
-License issues:
-Date: Mon, 10 Dec 2007 19:55:16 -0500
-From: TeamSpeak Sales <sales@tritoncia.com>
-To: 'Marc Weber' <marco-oweber@gmx.de>
-Subject: RE: teamspeak on nix?
-
-Yes, that would be fine.  As long as you are not renting servers or selling
-TeamSpeak then you are more than welcome to distribute it.
-
-Thank you,
+  License issues:
+  Date: Mon, 10 Dec 2007 19:55:16 -0500
+  From: TeamSpeak Sales <sales@tritoncia.com>
+  To: 'Marc Weber' <marco-oweber@gmx.de>
+  Subject: RE: teamspeak on nix?
```

Comment:
Could you move this to a separate .txt file and merely reference it here so that it doesn't get mangled by any future formatting?